src/scalap/scala/tools/scalap/scalax/rules/Rule.scala

```diff
@@ -164,7 +164,7 @@ trait Rule[-In, +Out, +A, +X] extends (In => Result[Out, A, X]) {
 
   /** ^-^(f) is equivalent to ^^ { b2 => b1 => f(b1, b2) }
    */
-  def ^-^ [B1, B2 >: A, C](f: (B1, B2) => C) = map { b2: B2 => b1: B1 => f(b1, b2) }
+  def ^-^ [B1, B2 >: A, C](f: (B1, B2) => C) = map { (b2: B2) => (b1: B1) => f(b1, b2) }
```

Comment:
This was the abuse of syntax that made me embrace the parens. What is the paren pun? Emparenthetic?

It made me emparenthetic to the syntax change.